src/storage/src/storage_state.rs

```diff
@@ -180,25 +198,36 @@ impl<'w, A: Allocate> Worker<'w, A> {
                         .exports
                         .insert(export.id, export.description.clone());
 
-                    // TODO(chae): will we need sink_uppers/since tracking done here??
+                    self.storage_state.sink_write_frontiers.insert(
+                        export.id,
+                        Rc::new(RefCell::new(Antichain::from_elem(
+                            mz_repr::Timestamp::minimum(),
+                        ))),
+                    );
 
                     crate::render::build_export_dataflow(
                         &mut self.timely_worker,
                         &mut self.storage_state,
                         export.id,
                         export.description,
                         export.resume_upper,
-                    )
+                    );
+
+                    self.storage_state.reported_frontiers.insert(
```

Comment:
why do we have these two frontiers?

Reply:
missed this comment before -- but I explained in person.  One tracks what we've gotten from the sink impl and the other tracks what we've reported back to the controller!